libraries/chain/include/eosio/chain/abi_serializer.hpp

```diff
@@ -98,7 +98,7 @@ struct abi_serializer {
 
    void add_specialized_unpack_pack( const string& name, std::pair<abi_serializer::unpack_function, abi_serializer::pack_function> unpack_pack );
 
-   static constexpr size_t max_recursion_depth = 32; // arbitrary depth to prevent infinite recursion
+   static constexpr size_t max_recursion_depth = 1024; // arbitrary depth to prevent infinite recursion, increased from 32 for develop-boxed branch
```

Comment:
OK, since this is internal to this file, its increase won't have unforeseen impacts to other parts of code.

Reply:
It is used everywhere the abi serializer is used.